activerecord/lib/active_record/migration/command_recorder.rb

```diff
@@ -164,6 +164,16 @@ def invert_remove_index(args)
       alias :invert_add_belongs_to :invert_add_reference
       alias :invert_remove_belongs_to :invert_remove_reference
 
+      def invert_change_column_default(args)
+        table, column, options = *args
+
+        unless options && options.is_a?(Hash) && options[:from] && options[:to]
```

Comment:
That might be because the test uses strings, try adding one more to test booleans just in case :)